parm/use_cases/met_tool_wrapper/RegridDataPlane/RegridDataPlane_multi_field_multi_file.conf

```diff
@@ -0,0 +1,106 @@
+[config]
+# List of applications to run - only RegridDataPlane for this case
+PROCESS_LIST = RegridDataPlane
+
+# time looping - options are INIT, VALID, RETRO, and REALTIME
+# If set to INIT or RETRO:
+#   INIT_TIME_FMT, INIT_BEG, INIT_END, and INIT_INCREMENT must also be set
+# If set to VALID or REALTIME:
+#   VALID_TIME_FMT, VALID_BEG, VALID_END, and VALID_INCREMENT must also be set
+LOOP_BY = INIT
+
+# Format of INIT_BEG and INT_END using % items
+# %Y = 4 digit year, %m = 2 digit month, %d = 2 digit day, etc.
+# see www.strftime.org for more information
+# %Y%m%d%H expands to YYYYMMDDHH
+INIT_TIME_FMT = %Y%m%d%H
+
+# Start time for METplus run - must match INIT_TIME_FMT
+INIT_BEG=2005080700
+
+# End time for METplus run - must match INIT_TIME_FMT
+INIT_END=2005080700
+
+# Increment between METplus runs (in seconds if no units are specified)
+#  Must be >= 60 seconds
+INIT_INCREMENT = 1M
+
+# List of forecast leads to process for each run time (init or valid)
+# In hours if units are not specified
+# If unset, defaults to 0 (don't loop through forecast leads)
+LEAD_SEQ = 3H
+
+# Order of loops to process data - Options are times, processes
+# Not relevant if only one item is in the PROCESS_LIST
+# times = run all wrappers in the PROCESS_LIST for a single run time, then
+#   increment the run time and run all wrappers again until all times have
+#   been evaluated.
+# processes = run the first wrapper in the PROCESS_LIST for all times
+#   specified, then repeat for the next item in the PROCESS_LIST until all
+#   wrappers have been run
+LOOP_ORDER = times
+
+# If True, run regrid_data_plane on observation data
+OBS_REGRID_DATA_PLANE_RUN = True
+
+# name of input field to process
+# if unset, OBS_VAR1_NAME will be used
+OBS_REGRID_DATA_PLANE_VAR1_INPUT_FIELD_NAME = APCP
+OBS_REGRID_DATA_PLANE_VAR2_INPUT_FIELD_NAME = TMP
+
+# level of input field to process
+# if unset, OBS_VAR1_LEVELS will be used
+OBS_REGRID_DATA_PLANE_VAR1_INPUT_LEVEL = L0
+OBS_REGRID_DATA_PLANE_VAR2_INPUT_LEVEL = L0
+
+# name of output field to create
+# if unset, OBS_VAR1_NAME and OBS_VAR1_LEVELS will be combined to generate an output field name
+OBS_REGRID_DATA_PLANE_VAR1_OUTPUT_FIELD_NAME = APCP_01
```

Comment:
Should there be a second output field name here, or does it just use the input name if not specified?